src/sentry/api/endpoints/organization_details.py

```diff
@@ -248,6 +297,55 @@ def validate(self, attrs):
                 )
         return attrs
 
+    def save_trusted_relays(self, incoming, changed_data, organization):
+        timestamp_now = (
+            datetime.utcnow().isoformat() + "Z"
+        )  # make it utc relative with set timezone
```

Comment:
Please use this instead:

```py
from datetime import datetime 
from pytz import utc

datetime.utcnow().replace(tzinfo=utc).isoformat()
```

Reply:
yes, that is how it was at the beginning but it formats it as yyy-mm-ddThh:mm:ss.ffff+00:00
which is correct but IMO it is nicer to show it like yyy-mm-ddThh:mm:ss.ffffZ

Reply:
Maybe there’s a flag to force the Z in isotimestamp, but I’d rather go with some built in API than manual string manipulation on a date time without time zone.

Reply:
ok.. as you prefer, I'll see if I can "convince it to use Z" and if not we'll go with `+00:00`